src/pages/studyView/table/ClinicalEventTypeCountTable.tsx

```diff
@@ -241,7 +253,20 @@ export default class ClinicalEventTypeCountTable extends React.Component<
         return {
             [ClinicalEventTypeCountColumnKey.CLINICAL_EVENT_TYPE]: 0,
             [ClinicalEventTypeCountColumnKey.COUNT]: 0,
-            [ClinicalEventTypeCountColumnKey.FREQ]: 0,
+            [ClinicalEventTypeCountColumnKey.FREQ]: correctMargin(
```

Comment:
what's this about?  (i see it isn't new)

Reply:
Added this to dynamically calculate the marge based on freq string.

This is similar to how MultiSelectionTable.tsx aligns the Freq column